enzyme/mkv.py

```diff
@@ -330,7 +338,7 @@ def fromXML(cls, xmlElement):
         :type xmlElement: :class:`xml.etree.ElementTree.Element`
 
         """
-        targets = Targets.fromXML(xmlElement.find('Targets')) if xmlElement.find('Targets') else Targets()
+        targets = Targets() if xmlElement.find('Targets') is None else Targets.fromXML(xmlElement.find('Targets'))
```

Comment:
Because of the order of execution of the statement, having `xmlElement.find('Targets') == None` would raise an error when `Targets.fromXML(...)` is evaluated. No what we want. Second version tests *before* evaluating.